inductiva/fluids/fluid_block/fluid_block.py

```diff
@@ -204,8 +209,10 @@ def _(cls, simulator: DualSPHysics):  # pylint: disable=unused-argument
 def _(self, simulator: DualSPHysics, input_dir):  # pylint: disable=unused-argument
     """Creates DualSPHysics simulation input files."""
 
-    replace_params_in_template(
-        template_path=os.path.join(os.path.dirname(__file__),
+    template_files_dir = os.path.join(SCENARIO_TEMPLATE_DIR,
+                                      DUALSPHYSICS_TEMPLATE_INPUT_DIR)
+    templates.replace_params_in_template(
```

Comment:
```
templates.replace_params
```
no need to have "_in_template". 
The in_template suffix is only needed when you import the function, thereby loosing all the context.
Don't change it now, because it would require changes in many places, but let's discuss this with @hpenedones and how to systematically do this sort of changes across all code.

Reply:
Yes, if we follow the Google python guidelines we won't be importing classe or methods directly, therefore you will have the module "templates" and will call `templates.replace_params` (like you just did), so there's no need for the redundancy in the method name... Cool